File: countevennumber.py
def countevennumber(arr):
    count = 0
    for i in arr:
        if i % 2 == 0:
            count += 1
            
    return count

n = int(input("Enter the number of elements in the array: "))
arr = list(map(int, input("Enter the elements of the array separated by space: ").split()))
x = countevennumber(arr)
print("The count of even numbers in the array is:", x)

File: countevennumber.java

public class countevennumber {

    public static void main(String[] args) {
        int count = 0;
        int arr[] = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
        for (int i : arr){
            if ( i % 2 == 0){
                count++;
            }
        
        }
        System.out.println("The count of even numbers in the array is: " + count);
    }
}